evadb/functions/function_bootstrap_queries.py

```diff
@@ -214,6 +214,30 @@
     EvaDB_INSTALLATION_DIR
 )
 
+Upper_function_query = """CREATE FUNCTION IF NOT EXISTS UPPER
+        INPUT  (input ANYTYPE)
+        OUTPUT (output ANYTYPE)
```

Comment:
Output type

Reply:
Done, I was under the impression it won't be needed since if the input is numeric, it will be a no-op.